pdns/backends/gsql/gsqlbackend.cc

```diff
@@ -1836,6 +1870,11 @@ void GSQLBackend::extractComment(const SSqlStatement::row_t& row, Comment& comme
   comment.content = row[5];
 }
 
+void GSQLBackend::extractZoneReference(const SSqlStatement::row_t& row, std::tuple<string, string>& reference)
```

Comment:
i guess this is here because `extractRecord` and `extractComment` exist? they are called from multiple places, but the one line of code here could live inline in `getSubZones`.